lib/modules/dosomething/dosomething_signup/includes/Signup.php

```diff
@@ -88,6 +88,22 @@ public static function find(array $filters = []) {
   private function build($data) {
     $this->id = $data->sid;
     $this->created_at = $data->timestamp;
+
+    $northstar_response = dosomething_northstar_get_northstar_user($data->uid);
```

Comment:
cause thats how we do it for reportbacks & reportback items https://github.com/DoSomething/phoenix/blob/09a6a60ae359882228bfed9b6b01cb8bb97f623e/lib/modules/dosomething/dosomething_reportback/includes/Reportback.php#L147